src/openqaoa-core/openqaoa/problems/vehiclerouting.py

```diff
@@ -377,7 +474,9 @@ def plot_solution(
         fig : matplotlib.pyplot.Figure()
             The graph visualization of the solution.
         """
-
+        pos = self.pos
+        if pos is []:
+            raise ValueError(f"There are not [x, y] coordinates of the VRP in pos: {pos}.")
         colors = colormaps["jet"] if colors is None else colors
```

Comment:
I noticed that colors does have a typing in the initialization parameter. Should we consider having a constraint on what colors can be put in? i.e. only allowing matplotlib colormaps(?)

I noticed that in line 500 the line reads: `tours_color[f"x_{i}_{j}"] = colors((vehicle + 1) / self.n_vehicles)`
Is there a particular input `color` typing that resolves this?

Reply:
@shahidee44 At the moment there are two ways to include colors, one takes values by default with a colormap "jet" and the second is providing colors. The second option must be a list of colors which gives flexibility the user to add the colors wanted. Giving a restriction to provide only colormaps in my opinion removes that flexibility.

Reply:
Hmmm I see. Makes sense! To make it easier for the users to identify what input the `colors` parameter can accept, we ought to add the typehinting to the definition of the method.

Reply:
done